spec/controllers/managements_controller_spec.rb

```diff
@@ -43,12 +68,34 @@
       end
     end
 
+    describe 'POST /managements/site_update' do
+      it 'works!' do
+        data = [{"var"=>"site.Neta1", "value"=>"Toro"}, {"var"=>"site.Neta2", "value"=>"Unit"}]
+        post :site_update, settings: data
+        expect(response.status).to eq(302)
+        expect(response).to redirect_to '/managements/site_setting'
+        settings = CustomSetting.unscoped.where("var like 'site.Neta%'")
+        expect(settings.length).to eq(2)
+      end
+    end
+
     describe 'GET /managements/custom_contents_setting' do
       it 'works!' do
         get 'custom_contents_setting'
         expect(response.status).to eq(200)
       end
     end
+
+    describe 'POST /managements/custom_contents_update' do
+      it 'works!' do
+        data = [{"var"=>"custom_content.Neta1", "value"=>"Toro"}, {"var"=>"custom_content.Neta2", "value"=>"Unit"}]
```

Comment:
__[rubocop]__
* Style/SpaceInsideHashLiteralBraces: Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceAroundOperators: Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceAroundOperators: Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceInsideHashLiteralBraces: Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/SpaceInsideHashLiteralBraces: Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceAroundOperators: Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceAroundOperators: Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceInsideHashLiteralBraces: Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)